pkg/application/create.go

```diff
@@ -116,10 +126,13 @@ func (svc *Service) handleSingleRepo(op CreateOp, localRepo *git.LocalRepository
 		return result, err
 	}
 
-	if err := localRepo.Push(git.PushOp{
-		Auth: op.GitAuth,
-	}); err != nil {
-		return result, err
+	log.Info().Msg("(git) pushing to remote")
+	if !op.DryRun {
+		if err := localRepo.Push(git.PushOp{
+			Auth: op.GitAuth,
+		}); err != nil {
+			return result, err
+		}
```

Comment:
Is monorepo case skipped intentionally? (can't leave comment there for some reason)

Reply:
I'm not sure which bit you mean

Reply:
I'm talking about `Service.handleMonorepo` method

Reply:
Is it expected that we still do the template rendering and file manipulation?

Reply:
Updated to skip templates in dry run mode